src/build/routes.ts

```diff
@@ -132,12 +132,12 @@ const models: TsoaRoute.Models = {
             "lastName_1": {"dataType":"string"},
             "lastName_2": {"dataType":"string"},
             "incorporationNumber": {"dataType":"string"},
-            "addressLine_1": {"dataType":"string","required":true},
+            "addressLine_1": {"dataType":"string"},
             "addressLine_2": {"dataType":"string"},
-            "city": {"dataType":"string","required":true},
-            "province": {"dataType":"string"},
-            "otherGeographicDivision": {"dataType":"string"},
```

Comment:
I think we don't technically know for sure if provinceLong is the same thing as otherGeographicalDivision (but it probably is?)

Reply:
Yeah, we haven't gotten a response yet but based on field length and contents I think this is a safe assumption at the moment